packages/xo-web/src/xo-app/hub/recipes/recipe-form.js

```diff
@@ -129,6 +139,21 @@ export default decorate([
         (_, { value }) =>
         sr =>
           sr.$pool === get(() => value.pool.id) && isSrWritable(sr),
+      versionList: async () => {
+        const res = await fetch('https://api.github.com/repos/kubernetes/kubernetes/releases')
+        if (res.ok) {
+          const rawList = await res.json()
+          const versionList = rawList
+            .filter(version => !version.tag_name.includes('-'))
```

Comment:
I think you can test `version.prerelease` here?

Reply:
I don't understand 🤔

Reply:
If I understand correctly, you want to filter out `rc` versions. And I think that's exactly what the property `prerelease` is there for.